rules_daml/daml.bzl

```diff
@@ -104,8 +104,16 @@ daml_compile = rule(
 
 def _daml_test_impl(ctx):
     script = """
-      {damlc} test --files {files}
-    """.format(damlc = ctx.executable.damlc.short_path, files = " ".join([f.short_path for f in ctx.files.srcs]))
+      set -eou pipefail
+
+      DAMLC=$(rlocation $TEST_WORKSPACE/{damlc})
```

Comment:
How come this doesn't require the usual [bash `rlocation` boilerplate](https://github.com/bazelbuild/bazel/blob/168ee7c589955372843115f67d9df2b4cc42e6ba/examples/shell/test.sh#L18)?

Reply:
I was also a bit surprised, but it's injected automatically - the difference here is that it's a Bazel's rule, not plain bash script used in `sh_test`. Generated test exec files does not contain any defintion of the `rlocation`, so execution of these tests outside of Bazel runtime will fail, but not sure if we care about such case (running tests generated by Bazel outside of Bazel).

Reply:
Interesting, that's good to know.

> not sure if we care about such case (running tests generated by Bazel outside of Bazel).

The environment between `bazel run` and `bazel test` is typically different, so I guess it's fine if these tests require `bazel test`. It's still possible to pass arguments using `--test_args` if that's necessary at some point.